tool/plugin/lib/edit.dart

```diff
@@ -69,6 +69,18 @@ List<EditCommand> editCommands = [
         'import com.android.tools.idea.gradle.dsl.model.BuildModelContext;',
     version: '4.1',
   ),
+  Subst(
+    path:
+        'flutter-studio/src/io/flutter/assistant/whatsnew/FlutterNewsBundleCreator.java',
+    initial: """
+    PluginManager pluginManager = PluginManager.getInstance();
+    IdeaPluginDescriptor descriptor = pluginManager.findEnabledPlugin(PluginId.getId("io.flutter"));
+""",
+    replacement: """
```

Comment:
Confused - how does this work when initial and replacement are the same?

Reply:
There are two hard-to-read differences. The first line is removed and "p" becomes "P" in the second.

Reply:
I completely misread, not sure how I missed the difference!